### ISSUE TITLE
Some strange errors in logs (TypeErrors)

### ISSUE THREAD

**Issue by xni** [Agoric/agoric-sdk#2620]
Hello! My running node publishes some weird things to logs:

```
Mar 11 21:43:50  ag-chain-cosmos[58777]: 2021-03-11T21:43:50.302Z SwingSet: kernel: Temporary logging of sent error (TypeError#1011)
Mar 11 21:43:50  ag-chain-cosmos[58777]: Nested error
Mar 11 21:43:50  ag-chain-cosmos[58777]:   TypeError#1011: target has no method "help", has ["addListener","connect","getLocalAddress","removeListener","revoke"]
Mar 11 21:43:50  ag-chain-cosmos[58777]:      at TypeError (packages/tame-metering/src/tame.js:130:18)
Mar 11 21:43:50  ag-chain-cosmos[58777]:   
Mar 11 21:43:50  ag-chain-cosmos[58777]:   TypeError#1011 ERROR_NOTE: Thrown from: (Error#1012) : 9013 . 0
Mar 11 21:43:50  ag-chain-cosmos[58777]:   TypeError#1011 ERROR_NOTE: Sent as error:liveSlots:v5#2
Mar 11 21:43:50  ag-chain-cosmos[58777]:   Nested error under TypeError#1011
Mar 11 21:43:50 ag-chain-cosmos[58777]:     Error#1012: Event: 9012.1
Mar 11 21:43:50  ag-chain-cosmos[58777]:        at Function.applyMethod (packages/tame-metering/src/tame.js:183:20)
Mar 11 21:43:50  ag-chain-cosmos[58777]:       at Object.deliver (packages/SwingSet/src/kernel/liveSlots.js:480:28)
Mar 11 21:43:50  ag-chain-cosmos[58777]:       at eval (packages/SwingSet/src/kernel/vatManager/deliver.js:51:48)
```

**Comment by michaelfig** [comment]
These logs are harmless, but I agree that they're weird and confusing.

We'll leave this issue open to track the silencing of development debug logs for validators.  This silencing should not happen for contract developers (using `agoric start`), since they're useful then.

**Comment by michaelfig** [comment]
This issue has been solved in the last software release with PR #2658.
